packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -13,13 +13,13 @@ A "floating" Promise is one that is created without any code set up to handle an
 Floating Promises can cause several issues, such as improperly sequenced operations, ignored Promise rejections, and more.
 
 This rule reports when a Promise is created and not properly handled.
-Valid ways of handling a Promise-valued statement include:
+Valid ways of silencing a Promise-valued statement include:
 
-- `await`ing it
-- `return`ing it
-- `void`ing it
-- Calling its `.then()` with two arguments
-- Calling its `.catch()` with one argument
+- `await`ing it (❌ unhandled)
+- `return`ing it (❌ unhandled)
+- [`void`ing it](#ignorevoid) (❌ unhandled)
```

Comment:
Maybe it would be good to call out here 
```suggestion
- [explicitly marking the promise as ignored by `void`ing it](#ignorevoid)
```

What do you think?

(and i personally would move it to the end of the list since it's an outlier in that it is just a way to disable the rule, not actually handle the promise, but up to you)

Reply:
I'm neutral to this suggestion. I mean, using "`void`ing" is coherent with "`await`ing" and "`return`ing" wording, so I'm not actually sure.

About moving to the bottom, this is perfectly fine. I like it.